indico/modules/events/registration/templates/management/reglist_filter.html

```diff
@@ -41,9 +42,19 @@
     <div class="section">
         <div class="icon icon-wrench"></div>
         <div class="toolbar right">
-            <a class="i-button js-reset-btn icon-close warning" type="button">
-                {%- trans %}Clear filters{% endtrans -%}
-            </a>
+            <div class="group left">
+                <a class="i-button icon-checkbox-checked arrow left icon-only" aria-hidden="true" href="#"
```

Comment:
The code for `.js-dropdown` ([global.js:119](https://github.com/indico/indico/blob/b1d94dc7f7cfb1bfca238f3ed4b7c481cfad7197/indico/htdocs/js/indico/jquery/global.js#L119)) is currently run only when the page is loaded . It can thus not be used for dialogs.

As for using it in `setupRegistrationList` to remove one line of code, it creates more problems and is not worth it. (There are several dropdowns sharing the same container for styling reasons and thus one widget must be created for multiple dropdowns.)